cpp/cologne.sh

```diff
@@ -9,9 +9,9 @@ readonly PASSWORD=$(inifile_lookup "$CONFIG_PATH" "" password)
 cd /usr/local/ub_tools/bsz_daten
 readonly newest=$(ls -t GesamtTiteldaten-??????.mrc | head -1)
 readonly date="${newest:17:6}"
-readonly output=IxTheoDatenFürKöln-"${date}".xml
+readonly output=IxTheoDatenFürKöln-"${date}".mrc
 generate_zeder_subset ixtheo koe Köln "${newest}" "${output}"
-rm --force /usr/local/vufind/public/docs/IxTheoDatenFürKöln-*.xml.7z
+rm --force /usr/local/vufind/public/docs/"${output}".7z
```

Comment:
The previous command removed all older files.

Reply:
That's why we have code reviews! ;-)